playbooks/roles/grafana/templates/grafana-renderer.service.j2

```diff
@@ -0,0 +1,36 @@
+[Unit]
+Description=Grafana Image Renderer container
+Wants=syslog.service
+
+[Service]
+Type=simple
+TimeoutSec=30s
+Restart=always
+User={{ grafana_os_user }}
+Group={{ grafana_os_group }}
+
+Environment=GRAFANA_RENDERER_IMAGE=grafana/grafana-image-renderer:2.0.0
```

Comment:
I know it's not everywhere this way, but what if we get rid of this additional line setting default image? We anyway force setting in the env file

Reply:
agree, deleted it from both service templates